### ISSUE TITLE
Capitalization typo in nuspec

### ISSUE THREAD

**Issue by teabaggs** [alexandrnikitin/Topshelf.Autofac#8]
The nuspec lists **TopShelf** dependency with a capital _S_. This causes some issues when other add on packages, Topshelf.Serilog for example, have a dependency on the same version of **Topshelf** with a lowercase _s_
```
   <dependencies>
      <dependency id="Autofac" version="[4.0.0,5.0.0)" />
      <dependency id="TopShelf" version="[4.0.1,5.0.0)" />
    </dependencies>
```

**Comment by alexandrnikitin** [comment]
Hi Philip, 
I'm sorry for the late reply, I was on vacation. Thank you for the issue and the fix.

**Comment by alexandrnikitin** [comment]
As I remember Topshelf was renamed from TopShelf few years ago. I didn't change the name then.
Pushed the fix as 3.1.1 https://www.nuget.org/packages/Topshelf.Autofac/3.1.1
